modules/etcd-defrag.adoc

```diff
@@ -7,6 +7,14 @@
 [id="etcd-defrag_{context}"]
 = Defragmenting etcd data
 
+For large and dense clusters, etcd can suffer from poor performance if the keyspace grows too large and exceeds the space quota. Periodically maintain and defragment etcd to free up space in the data store. Monitor Prometheus for etcd metrics and defragment it when required; otherwise, etcd can raise a cluster-wide alarm that puts the cluster into a maintenance mode that accepts only key reads and deletes.
+
+.Monitor these key metrics:
+
+* `etcd_server_quota_backend_bytes`, which is the current quota limit
+* `etcd_mvcc_db_total_size_in_use_in_bytes`, which indicates the actual database usage after a history compaction
+* `etcd_debugging_mvcc_db_total_size_in_bytes`, which shows the database size, including free space waiting for defragmentation
```

Comment:
Based on [SOP](https://github.com/openshift/ops-sop/blob/master/v4/troubleshoot/etcd.md#defragmenting-etcd-data), we also refer to the amount of space that can be freed after defragmentation with following query:
```bash
(etcd_mvcc_db_total_size_in_bytes - etcd_mvcc_db_total_size_in_use_in_bytes)/1024/1024
```
Gets handy when deciding if manual defragmentation required or not.

Reply:
Added that info to the PR, thanks!